dt/uni05epsilon/edpm-post-ceph/nodeset/resources/nova-custom.yaml

```diff
@@ -1,15 +1,3 @@
----
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: nova-custom-config
-data:
-  25-nova-custom.conf: |
-    [DEFAULT]
-    # Override our defaults in this dt to get coverage for metadata-api based
-    # cloud-init scenarios
-    force_config_drive = False
-
```

Comment:
Maybe since we're moving this block out and giving it its own resource file, we could rename this file to something specific as well, i.e. `nova-custom-dataplane-service.yaml`?  I was just initially confused why we have a generic `nova-custom.yaml` and I wonder if others might be too when looking through this.

Reply:
I agree, I aimed to rename things to be easier to understand, so I am happy for the suggestion.
So, there is now:
– [resources/nova-custom-config.yaml](https://github.com/sdatko/openstack-k8s-operators-architecture/blob/epsilon-ceph/dt/uni05epsilon/edpm-post-ceph/nodeset/resources/nova-custom-config.yaml)
– [nova-custom.yaml](https://github.com/sdatko/openstack-k8s-operators-architecture/blob/epsilon-ceph/dt/uni05epsilon/edpm-post-ceph/nodeset/resources/nova-custom.yaml) that will become **nova-custom-service.yaml**
Right?

Do I understand correctly that the service should remain as `nova-custom` in [values.yaml](https://github.com/sdatko/openstack-k8s-operators-architecture/blob/epsilon-ceph/examples/dt/uni05epsilon/values.yaml#L22C9-L22C20)? (Or you want me also to update the name [there](https://github.com/sdatko/openstack-k8s-operators-architecture/blob/epsilon-ceph/dt/uni05epsilon/edpm-post-ceph/nodeset/resources/nova-custom.yaml#L5)?)

Reply:
Correct, the `OpenStackDataPlaneService` can stay as-named.  It's just the file name that we might want to change to help indicate what it contains.

Reply:
Ok, thanks for clarification (day of learning! :1st_place_medal:)
I updated the file nova-custom.yaml -> nova-custom-service.yaml and the reference in kustomize.yaml